### PR TITLE
port blob

### PR DESCRIPTION
**Pull request by JohnnyDangerSpace**
## About the PR

ports blob from https://github.com/Rxup/space-station-14 & from goobstation 

credit to VigersRay for original code and
Roudenn and Rxup for maintaining
vladospupuos for nigthvision
jorgun for the goob port

## Why / Balance

its blob bro its literally tider defence and its awesome

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: jorgun VigersRay Roudenn Rxup vladospupuos, and fishbait
- add: Added Blob

### CODE REVIEW

**Comment by JohnnyDangerSpace** [comment]
poster 53 legit

**Comment by taydeo** [comment]
RSI fail unrelated. If you want it to be pretty, just delete poster53_legit.png from the folder.

File: Content.Shared/CCVar/CCVars.cs [hunk 1]
```diff
@@ -20,9 +20,78 @@ public sealed partial class CCVars : CVars
     public static readonly CVarDef<bool> DebugOptionVisualizerTest =
         CVarDef.Create("debug.option_visualizer_test", false, CVar.CLIENTONLY);
 
-    /// <summary>
-    /// Set to true to disable parallel processing in the pow3r solver.
-    /// </summary>
-    public static readonly CVarDef<bool> DebugPow3rDisableParallel =
-        CVarDef.Create("debug.pow3r_disable_parallel", true, CVar.SERVERONLY);
+        /// <summary>
```
Comment (taydeo) [inline Content.Shared/CCVar/CCVars.cs:23]: This should not be here. Delete all of this. Move the CVars related to Blob to their own folder. No changes should exist in this file.

**Review by taydeo — CHANGES_REQUESTED** [review]

File: Content.Shared/CCVar/CCVars.cs (hunk 1 above)
Comment (JohnnyDangerSpace) [reply Content.Shared/CCVar/CCVars.cs]: always gotta clean up goobs messes smh

Comment (taydeo) [reply Content.Shared/CCVar/CCVars.cs]: you say this but you still did it wrong

**Review by taydeo — APPROVED** [review]

**Comment by taydeo** [comment]
Test fail unrelated fuck it we ball

**Pull request merged**